### PR TITLE
[runtime-security] fix container_path and inode

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

This PR fix the container_path and the inode reported. This also allows to run the functional tests in docker.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Test have been added to check both container_path and inode

### CODE REVIEW

**Review by lebauce — APPROVED** [review]

**Review by Gui774ume — APPROVED** [review]
LGTM

File: pkg/security/ebpf/c/unlink.h [hunk 1]
```diff
@@ -45,18 +45,21 @@ SYSCALL_KPROBE3(unlinkat, int, dirfd, const char*, filename, int, flags) {
...
-    syscall->unlink.overlay_numlower = get_overlay_numlower(dentry);
+
+    // if second pass, ex: overlayfs, just cache the inode that will be used in ret
+    if (syscall->unlink.path_key.ino) {
+        syscall->unlink.real_inode = get_dentry_ino(dentry);
+        return 0;
+    }
+
     syscall->unlink.path_key.ino = get_dentry_ino(dentry);
+    syscall->unlink.overlay_numlower = get_overlay_numlower(dentry);
```
[the remark this answers is not in the document]
Comment (safchain) [reply pkg/security/ebpf/c/unlink.h:62]: Probably, I decided to ignore this for now and to address this in another PR. is it ok for you ?

[the remark this answers is not in the document]
Comment (Gui774ume) [reply pkg/security/ebpf/c/unlink.h:62]: Sure 👍  I was just curious

**Review by Gui774ume — APPROVED** [review]
LGTM

**Pull request merged**